lib/chef/chef_fs/data_handler/container_data_handler.rb

```diff
@@ -15,10 +15,10 @@ def preserve_key?(key)
           return key == "containername"
         end
 
-        def verify_integrity(object, entry, &on_error)
+        def verify_integrity(object, entry)
```

Comment:
I tend to agree we should do this with documentation. @lamont-granquist could you add some minimal YARDage to this PR for the things that used to take blocks?